### PR TITLE
Handler Errno::EHOSTDOWN error on Adapter init

### PR DESCRIPTION
**Pull request by lucasgomide**

### CODE REVIEW

File: lib/rails-cloud-tasks.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def queue_adapter
 
   def self.queue_adapter
     @queue_adapter ||= Adapter.new
-  rescue StandardError => e
+  rescue StandardError, Errno::EHOSTDOWN => e
```
Comment (guilhermearaujo) [inline lib/rails-cloud-tasks.rb:52]: Isn't `Errno::EHOSTDOWN` a subclass of `StandardError` already?

Comment (lucasgomide) [reply lib/rails-cloud-tasks.rb:52]: The exception is not captured

Comment (lucasgomide) [reply lib/rails-cloud-tasks.rb:52]: Is not. `Errno::EHOSTDOWN ` is an Exception raised by SO

https://www.honeybadger.io/blog/understanding-rubys-strange-errno-exceptions/

Comment (guilhermearaujo) [reply lib/rails-cloud-tasks.rb]: Then I don't know why rubocop didn't like it..

**Review by guilhermearaujo — APPROVED** [review]

**Pull request merged**